### DISCUSSION TITLE
How to check how many columns toggle is on

### DISCUSSION THREAD

**Discussion by bumpyy** [filamentphp/filament#7259] [Help]
I'm using the admin panel with simple/modal resource. let say each row have 'featured' column and i'm using toggle column to toggle it directly from table, but I want to make sure that there are no more than 3 featured records, how can i disable other row when i have 3 featured records or halt/cancel the save process and show notification.

thanks

**Comment by leandrocfe** [comment]
halt/cancel the save process

ManagePage.php
```php
use Filament\Pages\Actions\Action;
use Filament\Notifications\Notification;

Actions\CreateAction::make()
    ->before(function (array $data, Action $action) {

        //check / validate
        //you can use $data['field_name']
        //...

        //invalid
        Notification::make()
            ->title('Invalid...')
            ->danger()
            ->send();

        //close the modal
        $this->dispatchBrowserEvent('close-modal', ['id' => 'page-action']);

        //stop the proccess
        $action->halt();
    })
```

**Reply by dissto** [reply]
I dont know if you necessarily need to dispatch a browser event to close the modal. You could probably just cancel the action which would close the modal on its own.

```php

$action->cancel();

```

**Reply by bumpyy** [reply]
Yes, that work if i'm using modal, but i want to change it directly via "table" without using modal, is that possible ?
![image](https://github.com/filamentphp/filament/assets/28483765/375fa4e7-a358-4a00-b2bc-152199972135)
